CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## v 0.5.0 / 2017-11-14
+
+NOTE: This release breaks backward compatibility. `statsd_exporter` now uses
+a YAML configuration file. You must convert your mappings configuration to
+the new format when you upgrade.
```

Comment:
Should we add an example here? "If it looked like this before, it should now look like this, note that the name is not treated like a label anymore"?

Reply:
I was struggling what to say here. Linking to an example (or including here) may be the most helpful

Reply:
Something like:


For example, the configuration

    test.dispatcher.*.*.*
    name="dispatcher_events_total"
    processor="$1"
    action="$2"
    outcome="$3"
    job="test_dispatcher"

    *.signup.*.*
    name="signup_events_total"
    provider="$2"
    outcome="$3"
    job="${1}_server"

now has the format

```yaml
mappings:
- match: test.dispatcher.*.*.*
  help: "The total number of events handled by the dispatcher."
  name: "dispatcher_events_total"
  labels:
    processor: "$1"
    action: "$2"
    outcome: "$3"
    job: "test_dispatcher"
- match: *.signup.*.*
  name: "signup_events_total"
  help: "The total number of signup events."
  labels:
    provider: "$2"
    outcome: "$3"
    job: "${1}_server"
```

The help field is optional.

There is a [tool](https://github.com/bakins/statsd-exporter-convert) available to help with this conversion.